NEWS.md

```diff
@@ -1,5 +1,10 @@
 # pkgdown 1.1.0.9000
 
+* `build_logo()` now creates higher resolution favicons, which are better suited
```

Comment:
Can you please also remove magick from the DESCRIPTION?

House style is to not style package names specially (so please remove `` ` `` and `{`)